deploy/mariadb-cluster/templates/cluster.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: apps.kubeblocks.io/v1alpha1
+kind: Cluster
+metadata:
+  name: {{ include "kblib.clusterName" . }}
+  labels: {{ include "kblib.clusterLabels" . | nindent 4 }}
+spec:
+  clusterVersionRef: {{ .Values.version }}
+  terminationPolicy: {{ .Values.extra.terminationPolicy }}
+  {{- include "kblib.affinity" . | indent 2 }}
+  clusterDefinitionRef: mariadb # ref clusterdefinition.name
+  componentSpecs:
+    - name: mariadb
+      replicas: 1
+      componentDefRef: mariadb-compdef # ref clusterdefinition componentDefs.name
+      {{- include "kblib.componentMonitor" . | indent 6 }}
```

Comment:
did not see any monitoring config for mariadb. it's meanlingless to enable/disable the field.